File: C#/0344. Reverse String.cs
public class Solution {
    public void ReverseString(char[] s) {
        int left=0;
        int right=s.Length-1;
        while(left<right){
            char tmp=s[left];
            s[left]=s[right];
            s[right]=tmp;
            left++;
            right--;
        }
        
    }
}

File: archives/20190519python/0344. Reverse String.py
class Solution:
    def reverseString(self, s: List[str]) -> None:
        """
        Do not return anything, modify s in-place instead.
        """
        lo=0
        hi=len(s)-1
        while lo<hi:
            s[lo],s[hi]=s[hi],s[lo]
            lo+=1
            hi-=1
